aicsimageio/aics_image.py

```diff
@@ -138,58 +181,159 @@ def scenes(self) -> List[int]:
 
         >>> for i in range(len(image.scenes))
         """
-        pass
+        return self.reader.scenes
 
     @property
-    def current_scene(self) -> int:
+    def current_scene(self) -> str:
         """
         Returns
         -------
-        scene: int
+        scene: str
             The current operating scene.
         """
-        pass
+        return self.reader.current_scene
 
-    def set_scene(self, id: int):
+    @property
+    def current_scene_index(self) -> int:
+        """
+        Returns
+        -------
+        scene_index: int
+            The current operating scene index in the file.
+        """
+        return self.scenes.index(self.current_scene)
+
+    def set_scene(self, scene_id: str):
         """
         Set the operating scene.
 
         Parameters
         ----------
-        id: int
+        scene_id: str
             The scene id to set as the operating scene.
+
+        Raises
+        ------
+        IndexError: the provided scene id is not found in the available scene id list
+        """
+        # Only need to run when the scene id is different from current scene
+        if scene_id != self.reader.current_scene:
+
+            # Validate scene id
+            if scene_id not in self.scenes:
+                raise IndexError(
+                    f"Scene id: {scene_id} "
+                    f"is not present in available image scenes: {self.scenes}"
+                )
+
+            # Update current scene on the base Reader
+            # This clears the base Reader's cache
+            self.reader.set_scene(scene_id)
+
+            # Reset the data stored in the AICSImage object
+            self._xarray_dask_data = None
+            self._xarray_data = None
+            self._dims = None
+
+    def _transform_data_array_to_aics_image_standard(
+        self,
+        arr: xr.DataArray,
+    ) -> xr.DataArray:
+        # Determine if we include Samples dim or not
+        if dimensions.DimensionNames.Samples in arr.dims:
+            return_dims = dimensions.DEFAULT_DIMENSIONS_ORDER_WITH_SAMPLES
+        else:
+            return_dims = dimensions.DEFAULT_DIMENSION_ORDER
+
+        # Pull the data with the appropriate dimensions
+        data = transforms.reshape_data(
+            data=arr.data,
+            given_dims=self._known_dims or self.reader.dims.order,
+            return_dims=return_dims,
+        )
+
+        # Pull coordinate planes
+        coords = {d: arr.coords[d] for d in arr.coords if d in return_dims}
+
+        # Add channel coordinate plane because it is required in AICSImage
+        if dimensions.DimensionNames.Channel not in coords:
+            coords[dimensions.DimensionNames.Channel] = [None]
```

Comment:
Debatable how we want to handle this. I went with `[None]` because I encountered a valid OME file that had channels but no channel names which results in the base `OmeTiffReader` returning `[None, None`] for it's channel names. Consisteny seems good to me on that front.

Reply:
I didn't check but since I am simply pulling from `ome-types`: `OME.images[0].channels` or w.e. the path is. It just resulted in `[None, None]`. So to me that says that the channel names weren't attached.... which I didn't know was possible. I can investigate tonight.

Reply:
Ahhhhh it's because channel names aren't required for OME spec. Channel IDs are though.

Reply:
I think I will try to use channel name whenever it is present and if not I will use channel ID. Thoughts?

cc: @toloudis

Reply:
To keep to the OME spec I will also update the TiffReader to make fake channel names of: `f"Channel:{i}"`